### PR TITLE
Slona, Kartevie, and Ran task dialogue

### PR DESCRIPTION
**Pull request by allister-translator**
lemme know if i missed any

### CODE REVIEW

File: Translated/Dialog/Ran/ssn_012070.csv [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name_7006#0,"""Ran"""
+ssn_012070_0001#0,"""Greetings! Bruder told me that you wanted to about all the different ways of fighting on Halpha."""
```
Comment (DabirSA) [inline Translated/Dialog/Ran/ssn_012070.csv]: wanted to what?

File: Translated/Dialog/Ran/ssn_012070.csv [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+ssn_012070_0005#0,"""You will? Thank you so much!"""
+ssn_012070_0006#0,"""Out of all of the Operators at Command, I'm actually the newest one here. I want to catch up to the others as quickly as I can."""
+ssn_012070_0007#0,"""So in order to do that, I was thinking I'd first expand my knowledge of our enemies."""
+ssn_012070_0008#0,"""Therefore, I want to take measurements of the \""PSE Burst\"" phenomenon that's brought about by changes in the atmospheric PSE."""
+ssn_012070_0009#0,"""When a \""PSE Burst\"" occurs, it causes waves of enemies to descend upon the surrounding area."""
+ssn_012070_0010#0,"""Mt. Magnus has had an increase in activity lately, so could you go defeat some enemies there while I monitor the PSE fluctuations?"""
+ssn_012070_0101#0,"""Oh, I see... Let's speak at another time then, okay?"""
+ssn_012070_0201#0,"""If there's anything I can do for you as an Operator, simply say the word."""
+ssn_012070_0202#0,"""I hope to be able to make life easier for everyone to the best of my abilities."""
+ssn_012070_0301#0,"""This mission is surely going to be frought with peril. We'll be running support for you with everything we've got!"""
```
Comment (DabirSA) [inline Translated/Dialog/Ran/ssn_012070.csv]: fraught

File: Translated/Dialog/Ran/ssn_012090.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name_7418#0,"""Ran"""
+ssn_012090_0010#0,"""Thank you so much. I was able to get some amazing data thanks to you."""
+ssn_012090_0020#0,"""This sort of data collection is normally Liu Lin's specialty, but I want to be able to back her up when I can."""
+ssn_012090_0030#0,"""I'm going to work hard every day, so I can act to the benefit of everyone soon."""
```
Comment (DabirSA) [inline Translated/Dialog/Ran/ssn_012090.csv]: bit of a weird sentence here

File: Translated/Dialog/Ran/ssn_012090.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+ssn_012090_0040#0,"""It will ever continue to be a pleasure working with you!"""
+ssn_012090_0201#0,"""If there's anything I can do for you as an Operator, simply say the word."""
+ssn_012090_0202#0,"""I hope to be able to make life easier for everyone to the best of my abilities."""
+ssn_012090_0301#0,"""This mission is surely going to be frought with peril. We'll be running support for you with everything we've got!"""
```
Comment (DabirSA) [inline Translated/Dialog/Ran/ssn_012090.csv]: also fraught

File: Translated/Dialog/Ran/ssn_012090.csv [hunk 5]
```diff
@@ -0,0 +1,9 @@
+name_7418#0,"""Ran"""
+ssn_012090_0010#0,"""Thank you so much. I was able to get some amazing data thanks to you."""
+ssn_012090_0020#0,"""This sort of data collection is normally Liu Lin's specialty, but I want to be able to back her up when I can."""
+ssn_012090_0030#0,"""I'm going to work hard every day, so I can act to the benefit of everyone soon."""
+ssn_012090_0040#0,"""It will ever continue to be a pleasure working with you!"""
+ssn_012090_0201#0,"""If there's anything I can do for you as an Operator, simply say the word."""
+ssn_012090_0202#0,"""I hope to be able to make life easier for everyone to the best of my abilities."""
```
Comment (DabirSA) [inline Translated/Dialog/Ran/ssn_012090.csv]: weird sentence here

File: Translated/Dialog/Slona/ssn_012010.csv [hunk 6]
```diff
@@ -0,0 +1,12 @@
+name_7418#0,"""Slona"""
+ssn_012010_0001#0,"""Welcome to Central."""
+ssn_012010_0002#0,"""This is where many ARKS make their home. Of course, a great number of Meteorn as well."""
+ssn_012010_0003#0,"""There's a good range of facilities available as well, so definitely take a look around."""
+ssn_012010_0101#0,"""Bruder filled me in on the details. I've been waiting for you."""
+ssn_012010_0102#0,"""This is I'll tell you what I know."""
```
Comment (DabirSA) [inline Translated/Dialog/Slona/ssn_012010.csv]: This is what?

Comment (allister-translator) [reply Translated/Dialog/Slona/ssn_012010.csv]: fixed x_x

File: Translated/Dialog/Ran/ssn_012090.csv (hunk 5 above)
Comment (allister-translator) [reply Translated/Dialog/Ran/ssn_012090.csv]: fixed x_x

File: Translated/Dialog/Ran/ssn_012090.csv (hunk 4 above)
Comment (allister-translator) [reply Translated/Dialog/Ran/ssn_012090.csv]: fixed x_x

File: Translated/Dialog/Ran/ssn_012090.csv (hunk 3 above)
Comment (allister-translator) [reply Translated/Dialog/Ran/ssn_012090.csv]: fixed x_x

File: Translated/Dialog/Ran/ssn_012070.csv (hunk 2 above)
Comment (allister-translator) [reply Translated/Dialog/Ran/ssn_012070.csv]: フィックスド

File: Translated/Dialog/Ran/ssn_012070.csv (hunk 1 above)
Comment (allister-translator) [reply Translated/Dialog/Ran/ssn_012070.csv]: 直しましたでちゅ(´・ω・`)

**Pull request merged**